wx/lib/agw/aui/framemanager.py

```diff
@@ -5080,6 +5083,13 @@ def ClosePane(self, pane_info):
         if to_destroy:
             to_destroy.Destroy()
 
+        # Now inform the app that we closed a pane.
+        e = AuiManagerEvent(wxEVT_AUI_PANE_CLOSED)
+        e.SetPane(pane_info)
+        e.SetCanVeto(False)
+        e.SetId(wxEVT_AUI_PANE_CLOSED)
+        wx.CallAfter(self.ProcessMgrEvent, e)
```

Comment:
Is wx.CallAfter necessary here?

Reply:
Also, would it make sense to just use the `FireEvent` method to create and send the event instead of doing it all here?

Reply:
Not 100% sure, but i think i thought to have the closing event be finished before fireing the closed event. Anyway, i switch to `FireEvent` which in also uses `wx.CallLater`.